common/src/main/java/tds/common/web/interceptors/RestTemplateLoggingInterceptor.java

```diff
@@ -98,5 +115,13 @@ private void logResponse(final ClientHttpResponse response, final UUID traceId)
             response.getStatusText(),
             response.getHeaders(),
             bodyString.isEmpty() ? "<no body>" : bodyString);
+
+        try {
+            eventLogger.putField(RESPONSE_CODE.name(), response.getStatusCode());
+            eventLogger.putField(RESPONSE_CODE_TEXT.name(), response.getStatusText());
+        } catch (Exception ignored) {
+        }
+        eventLogger.setEndMillis(System.currentTimeMillis());
+        eventLogger.info(appId, request.getURI().getPath(), SERVICE_RETURN.name(), null, null);
```

Comment:
We touched on it offline, and plan to have a discussion Monday, but having two separate logging paths (eventLogger.info->logstash and log.info->stdout/logfile) feels a little confusing to me.